cli/local/docker_spec.go

```diff
@@ -134,7 +134,12 @@ func deployPythonContainer(api *spec.API, awsClient *aws.Client) error {
 			resources.Memory = api.Compute.Mem.Quantity.Value()
 		}
 		if api.Compute.GPU > 0 {
-			runtime = "nvidia"
```

Comment:
I believe that `runtime := ""` above and `Runtime:   runtime,` below can be deleted